apps/central/src/components/Groups.vue

```diff
@@ -204,7 +221,7 @@ export default {
         .then((data) => {
           this.schemas = data._schemas;
           this.loading = false;
-          if (this.hasManagerPermission) {
+          if (this.hasManagerPermission && this.showChangeColumn) {
             this.fetchChangelogStatus();
```

Comment:
if showChangeColumn is initially false and is changed to true later, doesn't the fetchChangeLogStatus also need to happen? Does this if need to be moved to the watch?